### ISSUE TITLE
Can't patch or post on instance

### ISSUE THREAD

**Issue by Sh4d1** [scaleway/scaleway-sdk-go#24]
Version: f1fb457

When posting or patching on instancem I got:
`received status code 400: error type is invalid_request_error: Request data is empty. `

**Comment by Sh4d1** [comment]
```
i := instance.NewApi(scwClient)
        if err != nil {
                log.Fatalf(err.Error())
        }
        request := &instance.CreateServerRequest{
                Zone: utils.ZoneFrPar1,
                Name: "test",
        }
        _, err := i.CreateServer(request)
        if err != nil {
                log.Fatalf(err.Error())
        }
```

**Comment by Sh4d1** [comment]
Fixed by #23
